resources/js/Cookies.js

```diff
@@ -32,12 +32,12 @@ class LaravelCookieConsent {
     }
 
     addScripts(data) {
-        if(!data.scripts) {
+        if (!data.scripts) {
             return;
         }
 
         data.scripts.forEach(script => {
-            const scriptRegex = /<script.*<\/script>/;
+            const scriptRegex = /<script([^]*)<\/script>/;
```

Comment:
Could you please explain why a capture group would be necessary and why a negated empty set (`[^]*`) is better than `.*`? Thanks :)